doc/classes/Object.xml

```diff
@@ -374,6 +374,7 @@
 			<argument index="1" name="value" type="Variant">
 			</argument>
 			<description>
+				Set property into the object, after next iteration of the physics cycle.
```

Comment:
While that's true when you're in the idle step (`_process`), it might be confusing if you're calling it in `_physics_process`, as you're then *during* the physics step, and the value will be changed after the idle step, so *before* the next physics cycle.

So something like "after the current frame's physics step" might be more accurate. Or "during idle time" as mentioned in the related `call_deferred`. Might be good to also link to the latter, with e.g. `This is equivalent to calling [method set] via [method call_deferred], i.e. [code]call_deferred("set", [property, value])[/code]`.

Reply:
I stumbled upon `call_deferred` after issuing this PR and wanted to double back and more closely match that docstring, so I'm glad you reference that as well. I'll make these changes when I have time later today. Thanks for the suggestions.